#! /bin/sh /usr/share/dpatch/dpatch-run
## 16_nodhcp.dpatch by Ulrich Dangel <mru@grml.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Handle explicit "nodhcp"

@DPATCH@

diff --git a/scripts/live b/scripts/live
index 5681362..46ef54b 100755
--- a/scripts/live
+++ b/scripts/live
@@ -88,6 +88,8 @@ Arguments ()
 
 			nodhcp)
				unset DHCP
+				NODHCP="Yes"
+				export NODHCP
 				;;
 
 			ethdevice=*)
